pom.xml

```diff
@@ -6,7 +6,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.5.2</version>
+		<version>2.7.10</version>
```

Comment:
Better use properties

Reply:
is it in any way possible to put the properties in the parent?

Reply:
Why not?